src/components/bubble/Bubble.stories.mdx

```diff
@@ -0,0 +1,117 @@
+import {ArgsTable, Meta, Story, Canvas} from '@storybook/addon-docs';
+import {StoryVariant, StoryVariantBorderBox} from '../../_docs/utils';
+import {Preview} from '../../_docs/blocks/';
+
+import Bubble, {DIRECTION, ALIGNMENT, BUBBLE_COLOR} from './Bubble';
+import BubbleA11y from './stories/Bubble.a11y.mdx';
+
+<Meta
+  title="Layout/Bubble"
+  component={Bubble}
+  argTypes={{
+    direction: {control: {type: 'select', options: DIRECTION}},
+    color: {control: {type: 'select', options: BUBBLE_COLOR}},
+    alignment: {control: {type: 'select', options: ALIGNMENT}},
+    full: {control: 'boolean'},
+    noShadow: {control: 'boolean'},
+    children: {
+      control: {disable: true},
+    },
+  }}
+  args={{
+    direction: DIRECTION.TOP,
+    alignment: ALIGNMENT.CENTER,
+    children: (
+      <span>
+        Hi there!!
+        <br />
+        Just wondering if you have any problems with your school work.
+      </span>
+    ),
+  }}
+/>
+
+# Bubble
+
+- [Stories](#stories)
+- [Accesibility](#bubble-accessibility)
```

Comment:
@domidomi why we need this special anchor (with additional `bubble` prefix?) I think main sections names should be constant between stories.

I am aware that  that this anchor is embedded in separate a11y story but main mdx should be responsible for defining main sections so we can move `accessibility` headline out of a11y.mdx file.

Reply:
I just followed format that Kasia has prepared. But you are right, we could move the accessibility headline to the main MDX!